# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/patch/no_ftello_fseeko.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./common.h.orig	Mon Aug 30 02:20:08 1999
+++ ./common.h	Wed Dec 29 09:54:31 1999
@@ -275,7 +275,7 @@
 #ifndef STDERR_FILENO
 #define STDERR_FILENO 2
 #endif
-#if HAVE_FSEEKO
+#if HAVE_FSEEKO && 0  /* This caused problems on big endian cpus */
   typedef off_t file_offset;
 # define file_seek fseeko
 # define file_tell ftello
